rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/30_limits.yml

```diff
@@ -161,3 +161,26 @@ setup:
                      ]))|\\[([^\\[\\]\\r\\\\]|\\\\.)*\\](?:(?:\\r\\n)?[\\t])*))*\\>(?:(?:\\r\\n)?[ \\t])*)(?:,\\s*( |
                      \".\\[\\]]))|\"(?:[^\\\"\\r\\\\]|\\\\.|(?:(?:\\r\\n)?[\\t]))*\"(?:(?:\\r\\n)?[ \\t])*)(?:\\.(?:( |
                      \\[\"()<>@,;:\\\\\".\\[\\]]))|\"(?:[^\\\"\\r\\\\]|\\\\.|(?:(?:\\r\\n)?[\\t]))*\"(?:(?:\\r\\n)?[\\t/"
+
+---
+"Prefix length limit":
```

Comment:
Lets add required version for this test, so it doesn't fail for tests with nodes < 8.15:

  - requires:
      cluster_features: ["gte_v8.15.0"]
      reason: "Limit for value in prefix query was introduced in 8.15"